patches/focus-trap-react+10.2.3.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/node_modules/focus-trap-react/index.d.ts b/node_modules/focus-trap-react/index.d.ts
+index 9274032..a5f633c 100644
+--- a/node_modules/focus-trap-react/index.d.ts
++++ b/node_modules/focus-trap-react/index.d.ts
+@@ -1,7 +1,8 @@
+-import { Options as FocusTrapOptions } from 'focus-trap';
++import { Options as FocusTrapOptions, FocusTargetOrFalse } from 'focus-trap';
+ import * as React from 'react';
+ 
+ export = FocusTrap;
++export type FocusTargetOrFalse = FocusTargetOrFalse;
```

Comment:
Is it required to export `FocusTargetOrFalse`? If not absolutely needed, let us avoid this.

Reply:
> The library doesn’t export this type

That's exactly the point. And I think it is intentional. I checked the documentation [here](https://github.com/focus-trap/focus-trap#createoptions) which is also explicitly referring to individual types and not `FocusTargetOrFalse`.  So, I think it would be better to abide by the documentation which is consistent with the expected usage. Does this make sense?